docs/02-app/01-building-your-application/01-routing/06-redirecting.mdx

```diff
@@ -252,6 +252,78 @@ export const config = {
 
 See the [Middleware](/docs/app/building-your-application/routing/middleware) documentation for more information.
 
+## External redirects
```

Comment:
```suggestion
<AppOnly>

## External redirects
```

Reply:
Is this necessary? I don't think this page isn't surfaced in Pages router.

![CleanShot 2024-01-24 at 08 52 35@2x](https://github.com/vercel/next.js/assets/446260/0d6236df-c063-4d07-95b1-010f3217e8f1)

![CleanShot 2024-01-24 at 08 52 44@2x](https://github.com/vercel/next.js/assets/446260/e598208e-c6cd-4498-9e05-cc7d871aee87)

Reply:
Disregard my comment. I see that #61083 starts sharing the page between routers.